locale/en.yml

```diff
@@ -27,7 +24,13 @@ fof-links:
       url_label: => fof-links.ref.url
       url_placeholder: => fof-links.ref.url
       use_rel_me: Add <code>rel="me"</code> attribute for identity verification on other sites
-      visibility: Link visibility
+      visibility:
+        help: Links by default are visible to only <code>admin</code> users. Adjust the permissions to specify who can see this link.
```

Comment:
I keep forgetting that the base groups have translations. However, because core allows them to be renamed I don't think translating in this way will be of great help. I do have an idea tho, I'll implement it..